e2b/compile_page.sh

```
@@ -0,0 +1,22 @@
+# This script runs during building the sandbox template
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)